### PR TITLE
open-cluster-management/backlog#10391 Add label mgd by

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
Hi @zhiweiyin318 . Here is an additional note about the new label in the GUI import section. Let me know if you have any thoughts. Thank you!

File: manage_cluster/import_gui.adoc [hunk 1]
```diff
@@ -41,7 +41,10 @@ You can import existing clusters from the {product-title} console for each of th
 . Click *Import an existing cluster*.
 . Provide a name for the cluster.
 By default, the namespace is used for the cluster name and namespace.
-. *Optional:* Add any _Additional labels_.
+. *Optional:* Add any _Additional labels_. 
++
+**Note:** If you import a Red Hat OpenShift Dedicated cluster and do not add a label for `vendor=OSD`, or if you add a label for `vendor=autodetect`, a `managed-by=platform` label is automatically added to the cluster to identify it as a Red Hat OpenShift Dedicated cluster.
```
Comment (zhiweiyin318) [inline manage_cluster/import_gui.adoc]: If you import a Red Hat OpenShift Dedicated cluster and do not add a label for `vendor=xxx`, or if you add a label for ~`vendor=autodetect`~ `vendor=auto-detect`,....

`vendor` is the label key, `xxx` is the label value, could be any string, I am not sure how to describe this meaning clearly.

**Comment by cadawson** [comment]
Thank you, @zhiweiyin318 . I made some changes, so see if this looks a little better. A question that I noticed: Is it "autodetect" or "auto-detect"? I noticed in the cli procedure, it was "auto-detect". I just want to make sure we are providing the correct wording. Thank you!

**Comment by zhiweiyin318** [comment]
@cadawson  sorry, my bad. it's `auto-detect`.

**Comment by cadawson** [comment]
No worries. It could been either, so I wanted to check. Thank you!

**Comment by zhiweiyin318** [comment]
/lgtm
thanks.

**Review by dockerymick — APPROVED** [review]
looks good to me!

**Pull request merged**